internal/provider/integration_aws_serverless_resource.go

```diff
@@ -279,20 +300,38 @@ func (r *integrationAwsServerlessResource) Schema(ctx context.Context, req resou
 								Optional:            true,
 								ElementType:         types.StringType,
 							},
+							"exclude_instance_ids_filter": schema.ListAttribute{
+								MarkdownDescription: "List of instance IDs to exclude.",
+								Optional:            true,
+								ElementType:         types.StringType,
+							},
+							"exclude_regions_filter": schema.ListAttribute{
+								MarkdownDescription: "List of regions to exclude.",
+								Optional:            true,
+								ElementType:         types.StringType,
+							},
+							"exclude_tags_filter": schema.MapAttribute{
+								MarkdownDescription: "Excluded Tags filter.",
+								Optional:            true,
+								ElementType:         types.StringType,
+							},
 							"ebs_volume_scan": schema.BoolAttribute{
 								MarkdownDescription: "Enable EBS volume scan.",
 								Optional:            true,
 							},
 							"ebs_scan_options": schema.SingleNestedAttribute{
-								Required: true,
+								Required:           true,
+								DeprecationMessage: "This field is deprecated and will be removed in the future.",
```

Comment:
Is not missing a flag here that indicates the argument is deprecated? Or just with this message it will detect it as deprecated?

Reply:
just by setting the message, there's no `deprecated` field